### ISSUE TITLE
Feature: Error Bar Disable Judgment

### ISSUE THREAD

**Issue by SimplyViper16** [dguzek/Simply-Love-SM5#408]
Hello!

I'm here to show you a feature I implemented and ask you if could be useful for the community.

### _**What is?**_ 🤔
You can disable the judgment from the error bar, that you don't want to track.
Here an example:

![ErrorBarExample](https://user-images.githubusercontent.com/10159143/177989131-959b175c-b945-4ec9-8b42-a0b525741b83.png)

### **_How to enable it?_**
In Player Options -> Advanced Options, after "Error Bar Options", I've added "_Error Bar Disable Judgment_", with different options based on the Game Mode.

**_ITG Mode_**
![ErrorBarDisableJudgmentITG](https://user-images.githubusercontent.com/10159143/177992076-9a57b5f3-9da5-428d-b06f-3cb39cc32f40.PNG)

**_FA+ Mode_**
![ErrorBarDisableJudgmentFAPlus](https://user-images.githubusercontent.com/10159143/177992125-f78c5f94-3eeb-408a-aad5-ff08c7a2bcdd.PNG)

The _disable judgment_ is applied for Colorful and Monochrome.
It also works for the emulated FA+ on ITG Mode.

What do you think about it?

**Comment by natano** [comment]
I like this feature! It looks useful for timing focused players.

Regarding the options: I suggest condensing this into one option "Only Fantastics + Excellents" and put it into the existing "Error Bar Options" row. Iirc, Waterfall also only shows fantastics and excellents and I'd guess that's what timing focused players would want.
What do you think, @Valentina16 and @teejusb?

Looking forward to your PR! :smile:

**Comment by SimplyViper16** [comment]
I guess we can add on the "Error Bar Options" but leaving to the player the choice of what show or not. I mean, taking myself as example, I use to play in FA+ mode with also the "great" active on the bar but Silver doesn't.
I'll open the PR asap so you can see what I've made (it's really hard to explain in few words) and waiting for all your enhancement!

Thank you!!

**Comment by teejusb** [comment]
I actually had a discussion about this exact thing in a discord server. I strongly agree that this should be a thing, but I actually don't think it's necessary to have an option row to select the windows. My rationale is that the people who will likely want to trim the error bar, will usually just want to cut it down to excellents so that it's more usable than having the huge great window. 

Like if you have all windows enabled, cutting off just the way off seems really niche.

I think having an "Error Bar Option" called "Trim" (or similar to that would be sufficient) that always cuts down the error bar to Fantastics + Excellents (+white fantastic for FA+ mode) is good.

Edit: I just read natano's suggestion which I think is essentially the same as mine so +1 to his comment.

**Comment by SimplyViper16** [comment]
So if I understand correctly this is the idea... right?
![image](https://user-images.githubusercontent.com/10159143/182450131-eb5083b9-bb9e-4939-a25a-df9737b5d29f.png)

**Comment by teejusb** [comment]
Yup that was the idea! In my head I'd consider just calling it "Trim" and edit the Languages text to mention that it trims it down to fantastics and excellents. I think it's a bit more concise, but not too set on it. @natano thoughts?

Also just to confirm, this option should also apply to the monochrome bar as well.

**Comment by SimplyViper16** [comment]
Sorry for my late reply, but I was on vacation and really needed to be away from development... But now here I am! 😃
I've made the edits you've suggested, so now there is an "Error Bar Option" called "Trim", which is mention into `[OptionExplanations]`.

I leave here some screenshots, so you can have a preview

_**Player Options**_
![image](https://user-images.githubusercontent.com/10159143/185890100-f7ba09f8-a8bf-4ac6-8e54-42ec2c8ca31b.png)

_**Gameplay**_
![ErrorBarExample2](https://user-images.githubusercontent.com/10159143/185893561-3a884ad2-7946-4da4-b5a7-746143d62578.png)

Waiting for your thoughts!
When it looks okay to you, I'll open the PR :)

**Comment by natano** [comment]
The screenshots look good to me and I'm fine with calling it "Trim".
cc @teejusb

**Comment by teejusb** [comment]
AHH I didn't respond to this but yes this looks good to me!
